test/Debugger/JsDebuggerAttachDetach.js

```diff
@@ -0,0 +1,16 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+eval("var x = '🇦';");
+
+function foo() {
+    var x = 1; /**bp:dumpSourceList();**/

```

Comment:
We should probably have this print something, I think we expect this to run on both detaches but the test doesn't currently validate that.